apps/root/src/common/utils/transaction-history/parsing.tsx

```diff
@@ -91,15 +93,14 @@ const parseDcaCreatedApiEvent: ParseFunction<DCACreatedApiEvent, DCACreatedEvent
       rate: {
         amount: BigInt(event.data.rate),
         amountInUnits: formatCurrencyAmount({ amount: BigInt(event.data.rate), token: dcaBaseEventData.fromToken }),
-        amountInUSD:
-          event.data.fromToken.price === null || isUndefined(event.data.fromToken.price)
-            ? undefined
-            : parseFloat(
-                formatUnits(
-                  BigInt(event.data.rate) * parseNumberUsdPriceToBigInt(event.data.fromToken.price),
-                  dcaBaseEventData.fromToken.decimals + 18
-                )
-              ).toFixed(2),
+        amountInUSD: isNil(event.data.fromToken.price)
```

Comment:
why was this change needed? isNil is going to return false if it is undefined as well